tests/e2e/module_file_manager/module_file_manager.cpp

```diff
@@ -0,0 +1,170 @@
+// Copyright 2022 Memgraph Ltd.
```

Comment:
I would add some cases where at least the error handling is checked, because it wouldn't be nice if because of a bug memgraph would crash in invalid cases.

Reply:
For this I need to update mgclient to support exceptions, so you'll see the changes tomorrow :smile: